packages/manager/apps/dedicated/client/app/network-security/traffic/traffic.controller.js

```diff
@@ -154,7 +154,10 @@ export default class TrafficController {
         after.setDate(after.getDate() - 14);
         break;
       case 'custom':
-        after.setTime(this.dateTime);
+        after = new Date(this.dateTime);
+        after = new Date(
+          after.getTime() + after.getTimezoneOffset() * 4 * 60000,
```

Comment:
I would recommend using date-fns to do this computation, it would be more understandable

You can also separate variables into constant

Personnaly I don't understand what 4 and 60 000 stand for

Reply:
When accessing the traffic dashboard via URL with a dateTime parameter, 
alerts were often not visible due to timezone misalignment. 
i just added proper timezone offset handling to ensure alerts appear within the 
displayed time window.

Reply:
I still don't understand what this `4` stands for

Reply:
This part of code is outdate, if you look to the new version of code you could see a constant variable which explain what is this `4`.